spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -157,14 +158,15 @@ class AbstractPopulationVertex(
     _SYNAPSE_BASE_N_CPU_CYCLES_PER_NEURON = 22
     _SYNAPSE_BASE_N_CPU_CYCLES = 10
 
-    # 5 elements before the start of global parameters
-    # 1. has key, 2. key, 3. n atoms, 4. n_atoms_peak 5. n_synapse_types
+    # Elements before the start of global parameters
+    # 1. has key, 2. key, 3. n atoms, 4. n_atoms_peak 5. n_colour_bits
```

Comment:
This change looked a bit strange to start off with until I realised that n_synapse_types wasn't in this list of elements any more (perhaps it used to be... ?)

Reply:
Yes, it was already a comment error; I checked that the correct number of elements was now added and just changed the mistake!